Makefile

```diff
@@ -46,3 +49,24 @@ test-bindings: $(PKG_BINDINGS_DIR)
 
 clean-bindings:
 	rm -rf $(PKG_BINDINGS_DIR)
+
+image: Dockerfile
+	DOCKER_BUILDKIT=1 \
+	$(DOCKER) build \
+	    --progress=plain \
+	    --tag $(BUILDIMAGE) \
+	    .
+
+docker-%: image
+	@echo "Running 'make $(*)' in docker container $(BUILDIMAGE)"
+	$(DOCKER) run \
+		--rm \
+		-e PWD \
+		-v $(PWD):$(PWD) \
+		-w $(PWD) \
+	    $(BUILDIMAGE) \
+			make $(*)
+
```

Comment:
The spacing here looks strange

Reply:
May be a tabs vs spaces thing. Will check.

Reply:
(Yes, was spaces instead of tabs).